loopchain/blockchain/blocks/v0_3/block_verifier.py

```diff
@@ -118,6 +127,19 @@ def verify_invoke(self, builder: 'BlockBuilder', block: 'Block', prev_block: 'Bl
                                      f"Expected({new_block.header.next_reps_hash}).")
             self._handle_exception(exception)
 
+        next_leader = new_block.header.next_leader
```

Comment:
The `next_leader` which get from parameter is not use and assigned from `new_block`.

Reply:
This code must be excluded.